.ruby-version

```diff
@@ -0,0 +1 @@
+2.3.0
```

Comment:
This only effects the development environemnt - doesn't get released with the gem